startupscript/dataproc/startup.sh

```diff
@@ -949,6 +961,53 @@ fi
     ${RUN_PYTHON} ${HAIL_SCRIPT_PATH}
   fi
 
+  #################################
+  # Configure Proxy Agent Overrides
+  #################################
+
+  # Map the CLI server to the appropriate UI url
+  if [[ "${TERRA_SERVER}" == *"verily"* ]]; then
+    # Map the CLI server to the appropriate UI url
+    if [[ "${TERRA_SERVER}" == "verily" ]]; then
+      ui_base_url="workbench.verily.com"
+    else
+      ui_base_url="${TERRA_SERVER/verily/terra}-ui-terra.api.verily.com"
+    fi
+  else
+    >&2 echo "ERROR: ${TERRA_SERVER} is not a known verily server."
+    exit 1
+  fi
+
+  # The banner.html file contains <style> wrapper tags and a series of CSS styles, and a set of html link elements that we want to modify.
+  # Begin banner.html modifications
+
+  # Insert a workspace link into the banner title
+  readonly WORKSPACE_LINK_EL='<a id="workspace" class="forum" target="_blank" href="https://'"${ui_base_url}/workspaces/${TERRA_WORKSPACE}"'"'">${TERRA_WORKSPACE}</a>"
+  sed -i 's#<banner-title>#<banner-title>\n'"${WORKSPACE_LINK_EL}"' \&gt; #' "${PROXY_AGENT_BANNER}"
+
+  # Add target blank property to all banner links so they open in a new tab
+  sed -i "s/class=\"forum\"/class=\"forum\" target=\"_blank\"/g" ${PROXY_AGENT_BANNER}
+
+  # Remove flex styling from the banner-account css class to prevent banner content from wrapping
+  sed -i '/banner-account {/,/}/{/flex:/d;/-ms-flex:/d;/-webkit-flex:/d;}' ${PROXY_AGENT_BANNER}
```

Comment:
I think it'd be preferred to use hashes to separate the substitution matching, since there's a lot of slashes in there. At the very least, the `${PROXY_AGENT_BANNER}` should be quoted.

Reply:
Done.